src/Sarif.Viewer.VisualStudio/ResultTextMarker.cs

```diff
@@ -57,478 +59,293 @@ public ResultTextMarker(int runId, Region region, string fullFilePath)
             Color = DEFAULT_SELECTION_COLOR;
         }
 
-        // This method is called when you click an inline link, with an integer target, which
+        /// <summary>
+        /// Attempts to navigate a VS editor to the text marker.
+        /// </summary>
+        /// <param name="usePreviewPane">Indicates whether to use VS's preview pane.</param>
+        /// <returns>Returns true if a VS editor was opened.</returns>
+        public bool TryNavigateTo(bool usePreviewPane)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            return this.TryNavigateTo(usePreviewPane, retryNaviation: true);
+
+        }
+
+        // This method is called when you click an in-line link, with an integer target, which
```

Comment:
>integer target, [](start = 73, length = 15)

Huh what integer target?

Reply:
Comment is absolutely no longer valid, and it never was.

---
In reply to: [481394177](https://github.com/microsoft/sarif-visualstudio-extension/pull/191#discussion_r481394177) [](ancestors = 481394177)